extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.schema.json

```diff
@@ -123,6 +123,105 @@
               "type": "integer",
               "meta:titleId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##title##88741",
               "meta:descriptionId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##description##46891"
+            },
+
+            "https://ns.adobe.com/experience/journeyOrchestration/messageID": {
+                "title": "MessageID",
+                "description": "Message ID referenced in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/publishedMessageID": {
+                "title": "PublishedMessageID",
+                "description": "Published message ID in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/interactionType": {
```

Comment:
If we know they can be either of "transactional" or "marketing", can we add these as enums ? That way you can ensure the right values flowing into this property and design your downstream flows accordingly

Reply:
Hey @mjasrotia , thanks for reviewing. We discussed it in our team and it does make sense to eventually make it enum, but for now it is too early and we would want to keep it like this for sometime as eventually the naming might be a bit different when hardened and more values could be added.

Reply:
@mukullamba04 , I understand that but please be aware of the fact that any enums added after the schemas are released and data is ingested could be a breaking change and that is not allowed

Reply:
> @mukullamba04 , I understand that but please be aware of the fact that any enums added after the schemas are released and data is ingested could be a breaking change and that is not allowed

As discussed with @mjasrotia , we will be sending test data for now and will change data type, if we do, before ingesting live customer data.